src/app/externalLinkHandlers.ts

```diff
@@ -30,6 +31,32 @@ export function isExternalLink(url: string) {
 	return !isInternalLink(url)
 }
 
+/**
+ * Try to extract Talk route from the link
+ * @param link - URL link
+ */
+function tryExtractTalkRoute(link: string) {
+	const url = new URL(link)
+
+	// Not a link to this instance
+	if (!appData.serverUrl || url.origin !== appData.serverUrl) {
+		return null
+	}
+
+	// Path without /index.php
+	const pathname = url.pathname.slice(url.pathname.startsWith('/index.php') ? '/index.php'.length : 0)
+
+	// Check whether the path is a valid Talk path or some other app
+	// eslint-disable-next-line prefer-regex-literals
+	const talkPathRe = new RegExp('^/apps/spreed(?:/(?:not-found|forbidden|duplicate-session))?|/call/[^/]+$')
```

Comment:
Is there a specific reason to not define it:
- outside of function?
- via constructor?